README.md

```diff
@@ -26,6 +26,9 @@ The Buildah package provides a command line tool that can be used to
 
 **[Tutorials](docs/tutorials)**
 
+**[Contributing](CONTRIBUTING.md)**
```

Comment:
I'd suggest just adding this line, dropping the next two and then alphabetize this after the Changelog at line 20.